pkgs/development/ocaml-modules/digestif/default.nix

```diff
@@ -12,6 +12,9 @@ buildDunePackage rec {
     sha256 = "09g4zngqiw97cljv8ds4m063wcxz3y7c7vzaprsbpjzi0ja5jdcy";
   };
 
+  # Fix tests with alcotest ≥ 1
+  patches = [ ./test_conv_ml.patch ];
```

Comment:
You could use `fetchpatch` here: <https://github.com/mirage/digestif/commit/8f69623ecccbd6e18cad97e42124615d9a771179>

Reply:
Done.